### ISSUE TITLE
discover archetypes no longer works remotely

### ISSUE THREAD

**Issue by gidden** [cyclus/cyclist2#99]
I assume this is due to updated remote execution changes, correct @yarden-livnat? @FlanFlanagan, do you have any ideas here?

**Comment by gidden** [comment]
to note, previously, if a server was selected (i.e., if remote was selected) discover archetypes would ping the server with a `cyclus -a` call.

**Comment by yarden-livnat** [comment]
Yes. I’ll fix it.

**Comment by yarden-livnat** [comment]
Fixed.

I noticed that Cycic checked at the start if a local server exists and select it as the default if it does. The new UI saves and restore the list of servers the user used as well as the last one used. As such I think we don’t need that capability.

For clarity:
The new interface allows you to switch between known servers (the default cycrun.fuelcycle.org and ‘local’). 
add: select the ‘— add new —‘ entry and edit it
remove: clear the entry (blank)
